components/home/__tests__/homepage-content.test.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+import { render } from "@testing-library/react";
+import { HomePageContent } from "../homepage-content";
+
+describe("homepage content render correctly", () => {
+  const { container } = render(<HomePageContent />);
+
+  it("renders correctly", () => {
+    expect(container.firstChild).toHaveClass("flex-1");
```

Comment:
Please perform more meaningful assertions that's obvious to the user experience other than merely asserting this `flex-1` class.